package/com.unity.formats.usd/Runtime/InitUsd.cs

```diff
@@ -61,12 +61,12 @@ public static bool Initialize()
             {
                 Debug.LogException(ex);
                 analyticsTimer.Stop();
-                UsdEditorAnalytics.SendUsageEvent(false, analyticsTimer.ElapsedMilliseconds);
+                UsdEditorAnalytics.SendUsageEvent(false, (float)analyticsTimer.Elapsed.TotalMilliseconds);
```

Comment:
Is there a reason why we cast this to float instead of just sending through the original double value?